OpenRA.Game/SelectableExts.cs

```diff
@@ -62,10 +62,7 @@ static int BaseSelectionPriority(ISelectableInfo info, Modifiers modifiers)
 
 		public static Actor WithHighestSelectionPriority(this IEnumerable<ActorBoundsPair> actors, int2 selectionPixel, Modifiers modifiers)
 		{
-			if (!actors.Any())
-				return null;
-
-			return actors.MaxBy(a => CalculateActorSelectionPriority(a.Actor.Info, a.Bounds, selectionPixel, modifiers)).Actor;
+			return actors.MaxByOrDefault(a => CalculateActorSelectionPriority(a.Actor.Info, a.Bounds, selectionPixel, modifiers)).Actor;
```

Comment:
```suggestion
			return actors.MaxByOrDefault(a => CalculateActorSelectionPriority(a.Actor.Info, a.Bounds, selectionPixel, modifiers))?.Actor;
```

Is probably needed.

Reply:
The elements of `actors` are `ActorBoundsPair` which is a struct, so `default(ActorBoundPair).Actor` is fine (and returns null).

Reply:
Ah, thanks for the explanation.